doc/default.nix

```diff
@@ -2,6 +2,7 @@
 let
   inherit (pkgs) lib;
   inherit (lib) hasPrefix removePrefix;
+  fs = lib.fileset;
```

Comment:
@infinisil, I've seen this `fs = lib.fileset` idiom a couple times in your patches now -- interested in why you're doing it. `fs.unions`, `fs.fileFilter` don't look like better names to me than `fileset.unions` or `fileset.fileFilter`. I'm not strongly opposed or have something else to suggest, just that the choice doesn't yet make me go "heck yes."

Reply:
Yeah that's fair, I think it's just smaller and I'm kind of used to it by now. It's like a qualified import (`use foo::bar as fb` in Rust, or `import qualified Foo.Bar as FB` in Haskell)

Reply:
I'd say let's keep it this way though, it's not a problem (unlike the `with` pattern!)

Reply:
Sure thing. Side note: I'm a filesystems engineer by trade, and so fs _always_ means file system to me ;-)